### DISCUSSION TITLE
入门指南

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#764] [General]
# 入门指南

了解如何开始使用 HB。

https://zh-hans.hbstack.dev/docs/getting-started/

**Comment by BG7ZDQ** [comment]
能感觉到这个框架很好，但是我真的看不懂文档，有点难懂..........

**Reply by razonyang** [reply]
对于新手来说学习曲线确实有点陡，如果有不懂的可以随意提问。

**Reply by BG7ZDQ** [reply]
谢谢。

它的使用方法和安装主题一样吗？
就是先建立一个站点，然后在themes文件夹中放入框架？

**Reply by razonyang** [reply]
这个不用将框架放入 themes 目录，因为其是模块化的，建议按照https://hbstack.dev/docs/getting-started/installation/ 的步骤进行安装，示例站点应该可以快速让你快速上手，另外示例站点的配置一般都带有注释说明，可以查看并修改。
